pkg/console/operator/sync_v400.go

```diff
@@ -408,3 +410,31 @@ func getDeploymentGeneration(co *consoleOperator) int64 {
 	}
 	return deployment.Generation
 }
+
+func copyCustomLogoConfigMap(co *consoleOperator, operatorConfig *operatorv1.Console) {
```

Comment:
I would follow the pattern we have set:

```golang
func SyncCustomLogoConfigMap(co *consoleOperator, operatorConfig *operatorv1.Console) (*corev1.ConfigMap, bool, error) {
```
Caller can ignore the `configmap`, but `errors` should bubble.